src/libraries/Microsoft.PowerFx.Core/Public/Values/TableValue.cs

```diff
@@ -136,6 +137,18 @@ private static ErrorValue NotImplemented(IRContext irContext, [CallerMemberName]
         public virtual async Task<DValue<RecordValue>> AppendAsync(RecordValue record, CancellationToken cancellationToken)
         {
             return DValue<RecordValue>.Of(NotImplemented(IRContext));
+        }
+
+        /// <summary>
+        /// Used to append a record to a table when baseRecord contains the primary key and updateRecord contains the fields to update.
+        /// </summary>
+        /// <param name="baseRecord"></param>
+        /// <param name="updateRecord"></param>
+        /// <param name="cancellationToken"></param>
+        /// <returns></returns>
+        public virtual async Task<DValue<RecordValue>> AppendAsync(RecordValue baseRecord, RecordValue updateRecord, CancellationToken cancellationToken)
```

Comment:
>AppendAsync [](http://example.com/codeflow?start=55&length=11)

We discussed offline - this is probably not needed

Reply:
Removed.